src/objects/button/button.scss

```diff
@@ -0,0 +1,81 @@
+@use "../../design-tokens/colors.yml";
+@use "../../design-tokens/motion.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/focus";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * Button object
+ *
+ * 1. More overtly actionable cursor appearance. Recently this tweak has come
+ *    with some contention, but our experience has been that the average user
+ *    doesn't differentiate cursor styles between different styles of action.
+ */
+
+.o-button {
+  background: colors.$primary-brand;
+  border-color: colors.$primary-brand-dark;
+  border-radius: sizes.$radius-medium;
+  border-style: solid;
+  border-width: sizes.$edge-medium;
+  color: colors.$text-light-emphasis;
+  cursor: pointer; /* 1 */
+  display: inline-block;
+  font: inherit;
```

Comment:
If we're inheriting `font` should we explicitly set `font-style` to `normal`? As is, if a button was in an italic block for some reason it would have italic text